flashlight/fl/test/tensor/jit/JitNodeTest.cpp

```diff
@@ -129,6 +101,8 @@ TEST(JitNodeTest, getSetResult) {
   ASSERT_TRUE(node->getResult().has_value());
   ASSERT_TRUE(allClose(node->getResult().value(), tensor));
   ASSERT_THROW(node->setResult(tensor.copy()), std::invalid_argument);
+  // "free" node
```

Comment:
```suggestion
// node is manually heap allocated
```

Reply:
Actually they are all heap allocated, but these ones didn't transition to the automated shared ownership, so they must be deleted manually. Documented.